File: Graph/adjMatrix.cpp
#include <bits/stdc++.h>
using namespace std;



int main(){

    cout << "Enter number of vertex in the graph" << endl;
    int V;
    cin >> V;

    vector<int>row(V,0);
    vector< vector<int> > edges(V,row);   
    
    cout << "Enter number of edges in the graph" << endl;
    int E;
    cin >> E;

    int sourceNode, destinationNode;
    cout << "Enter source node and vertext node of each edges" << endl;
    for(int i=0; i<E; i++){
        cin >> sourceNode >> destinationNode;
        edges[sourceNode][destinationNode] = 1;
        edges[destinationNode][sourceNode] = 1;
    }
    
    for(int i=0; i<edges.size(); i++){
        for(int j=0; j<edges[i].size(); j++){
            cout << edges[i][j] << " ";
        }cout << endl;
    }

    return 0;
}

File: Graph/Java/adjMatrix.java
import java.util.*;
class adjMatrix{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter number of vertices");
        int V = sc.nextInt();
        System.out.println("Enter number of Edges");
        int E = sc.nextInt();

        // For un-directed graph... // 0 --- based vertex   for 1 based vertex intialize the array of size V+1.
        int unDirectedAdjMatrix[][] = new int[V][V];

        // For directed graph....
        // 0 --- based vertex   for 1 based vertex intialize the array of size V+1.
        int directedGraphAdjMatrix[][] = new int[V][V];


        for(int i=0; i<E; i++){
            int u,v;
            u = sc.nextInt();
            v = sc.nextInt();

            // Un-directed graph...
            unDirectedAdjMatrix[u][v] = 1;
            unDirectedAdjMatrix[v][u] = 1;

            // Directed graph...
            directedGraphAdjMatrix[u][v] = 1;
        }

        for(int arr[]:unDirectedAdjMatrix){
            for(int k:arr){
                System.out.print(k + " ");
            }System.out.println();
        }

        System.out.println("++++++++++++++");
        
        for(int arr[]:directedGraphAdjMatrix){
            for(int k:arr){
                System.out.print(k + " ");
            }System.out.println();
        }
    }
}